src/main/kotlin/presentation/ui/ClothesSuggesterByCityNameCli.kt

```diff
@@ -0,0 +1,85 @@
+package presentation.ui
+
+import kotlinx.coroutines.*
+import org.damascus.domain.model.Cloth
+import org.damascus.domain.usecase.GetWeatherUseCase
+import org.damascus.domain.usecase.SuggestClothesUseCase
+import org.damascus.presentation.io.ConsoleDisplay
+import presentation.TerminalColor
+import presentation.io.InputReader
+import presentation.withStyle
+
+class ClothesSuggesterByCityNameCli(
+    private val printer: ConsoleDisplay,
+    private val inputReader: InputReader,
+    private val getWeatherUseCase: GetWeatherUseCase,
+    private val suggestClothesUseCase: SuggestClothesUseCase,
+) : UILauncher {
+
+    private var suggestedClothes: List<Cloth>? = null
```

Comment:
This variable does nothing, just declared to get assigned later and no use after.